dbt_macros/shared/all_evm_chains.sql

```diff
@@ -13,7 +13,8 @@
         'zksync', 
         'zora',
         'scroll',
-        'mantle'
+        'mantle',
+        'tac',
```

Comment:
what's the intention of adding this here, related to the PR scope of adding to dex trades?
this shouldn't have an impact. if we want to add for other use cases, should do in separate PR (although i'm not sure this helps anything in current setup?)